sumo-api/sumo/metrics/utils.py

```diff
@@ -173,6 +173,20 @@ def matrix_to_euler(matrix):
 # Precision recall curves
 #-------------------------------------
 
+def compute_ap(det_matches, det_scores, n_gt, recall_samples=None, interp=False, area_under_curve=True):
+    """
+    Compute average precision and precision-recall curve
+
+    Inputs:
+        area_under_curve: If true, compute average precision as area under the curve
```

Comment:
Please use the established documentation format and include the return documentation

Reply:
I can add the returns documentation. Didn't want to include more in the description here since it would be a copy paste of both of the subfunctions's documentation.